templates/kops/kops-private-topology.yaml.gotmpl

```diff
@@ -201,24 +199,42 @@ spec:
   kubernetesVersion: {{ getenv "KUBERNETES_VERSION" }}
   masterPublicName: api.{{ getenv "KOPS_CLUSTER_NAME" }}
   networkCIDR: {{ getenv "KOPS_NETWORK_CIDR" "172.20.0.0/16" }}
+  {{- if getenv "KOPS_CONFIGURED_VPC_ID" }}
+  networkID: {{ getenv "KOPS_CONFIGURED_VPC_ID" }}
+  {{- end }}
   networking:
     calico: {}
   nonMasqueradeCIDR: {{ getenv "KOPS_NON_MASQUERADE_CIDR" "100.64.0.0/10" }}
   sshAccess:
   - 0.0.0.0/0
   subnets:
   {{- range $index, $value := (getenv "KOPS_AVAILABILITY_ZONES" | strings.Split ",") }}
-  {{- $privateSubnets := getenv "KOPS_PRIVATE_SUBNETS" }}
+  {{- $privateSubnets := getenv "KOPS_PRIVATE_SUBNETS" | required "KOPS_PRIVATE_SUBNETS must be defined" }}
+  {{- $privateSubnetIDs := getenv "KOPS_CONFIGURED_PRIVATE_SUBNET_IDS" }}
```

Comment:
So the difference here is `CONFIGURED` means they won't be created? Would `EXISTING` maybe be more clear?

Reply:
Yes, means it won't be created or managed. If you don't like "configured"  how about "shared"?

Reply:
`SHARED` wfm